lua/core/bufferline.lua

```diff
@@ -1,27 +1,2 @@
 vim.api.nvim_set_keymap("n", "<S-l>", ":BufferNext<CR>", { noremap = true, silent = true })
 vim.api.nvim_set_keymap("n", "<S-h>", ":BufferPrevious<CR>", { noremap = true, silent = true })
-vim.api.nvim_set_keymap("n", "<leader>c", ":BufferClose<CR>", { noremap = true, silent = true })
```

Comment:
we need this keybinding to close the buffers, I think it got lost 😢

Reply:
It's a duplicate mapping.  Line 67 of whichkey already had this mapping